dash-renderer/src/components/core/Toolbar.react.js

```diff
@@ -33,7 +33,7 @@ function UnconnectedToolbar(props) {
                 },
                 styles.parentSpanStyle
             )}
-            onClick={() => dispatch(undo())}
+            onClick={() => dispatch(undo)}
```

Comment:
I know we do this in lots of places but re-rendering with a new function can in theory have an impact on performance and trigger GC more often. Since this is a functional component, could be wrapped in the `useCallback` hook which is a memoization hook offered by React since `16.6`.

https://reactjs.org/docs/hooks-reference.html#usecallback